tools/RAiDER/losreader.py

```diff
@@ -607,9 +607,9 @@ def get_radar_pos(llh, orb, out="lookangle"):
     num_iteration = 30
     residual_threshold = 1.0e-7
 
-    # Get xyz positions of targets here
+    # Get xyz positions of targets here from lat/lon/height
     targ_xyz = np.stack(
-        lla2ecef(llh[:, 1], llh[:, 0], llh[:, 2]), axis=-1
+        lla2ecef(llh[:, 0], llh[:, 1], llh[:, 2]), axis=-1
```

Comment:
This flipped. Can you make sure you are not introducing a bug. @jlmaurer @piyushrpt can you review.